examples/cli.rs

```diff
@@ -476,23 +499,25 @@ async fn run<C: Store + MessageStore>(subcommand: Cmd, config_store: C) -> anyho
             }
         }
         Cmd::GetGroup { group_master_key } => {
+            let( master_key, key ) = group_master_key;
             let manager = Manager::load_registered(config_store)?;
-            let group = manager.get_group_v2(group_master_key).await?;
+            let group = manager.get_group_v2(master_key).await?;
             println!("{:#?}", DebugGroup(&group));
             for member in &group.members {
                 let profile_key = base64::encode(&member.profile_key.bytes);
                 println!("{member:#?} => profile_key = {profile_key}",);
             }
+            manager.save_group(group, key.as_bytes().to_vec())?;
```

Comment:
I'm pretty sure you can get the bytes representation from the `GroupMasterKey` object directly as well, also without allocating a new `Vec`.